resources/views/auth/user_profile.blade.php

```diff
@@ -0,0 +1,67 @@
+@extends('layouts.main_layout')
```

Comment:
What's the purpose of this? it's called user_profile but it looks like a copy pasta of the login page?